bentoml/_internal/configuration/default_configuration.yaml

```diff
@@ -40,10 +40,13 @@ runners:
 
 tracing:
   type: zipkin
-  sample_rate: 0.0
+  sample_rate: Null
   excluded_urls: Null
   zipkin:
     url: Null
   jaeger:
     address: Null
     port: Null
+  otlp:
+    protocol: Null
+    url: Null
```

Comment:
Is it possible that you can change `Null` to ~ instead?
https://symfony.com/doc/current/components/yaml/yaml_format.html#nulls
imo it looks a bit nicer 😄

Reply:
That’s not specific to this PR, we can revisit later